profile-widget/divepixmapitem.cpp

```diff
@@ -108,9 +109,9 @@ void DivePictureItem::setFileUrl(const QString &s)
 
 void DivePictureItem::hoverLeaveEvent(QGraphicsSceneHoverEvent*)
 {
-	Animations::scaleTo(this, 0.2);
+	Animations::scaleTo(this, 0.2, qPrefDisplay::animation_speed());
```

Comment:
I think you got the order of arguments wrong (at least I get a compiler warning), should be: this, speed, 0.2

Reply:
Ooops. Good catch, thanks! Will fix later.

Reply:
Should be fixed now.